packages/era/test/e2e-mainnet/era.readwrite.integration.test.ts

```diff
@@ -23,31 +23,14 @@ describe.runIf(!process.env.CI)("read original era and re-write our own era file
     const SPR = SLOTS_PER_HISTORICAL_ROOT;
     const stateSlot = expectedEra * SPR;
 
-    const startTime = Date.now();
-    let t0 = startTime;
-
-    console.log("stage: open and read original era file");
```

Comment:
not against having these logs here if they were useful for your testing, also since we only run them manually, otherwise you can also take a look at `testLogger` we use in many tests